### DISCUSSION TITLE
bind custom mod combinations to number keys

### DISCUSSION THREAD

**Discussion by tovc** [ppy/osu#22131] [Ideas] [answered]
being able to activate custom mod combinations with number keys would make switching between 1.1x DT and 1.2x DT, for example, much easier. it would also be consistent with enabling mods with the alphabetical keys

![osu_2023-01-11_15-15-22](https://user-images.githubusercontent.com/49025619/211742486-357c3cab-faee-4a40-9b74-b366f324b473.png)

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/discussions/19683
